dotcom-rendering/src/model/badges.ts

```diff
@@ -0,0 +1,238 @@
+import type { FEBadgeType, FESpecialBadgeType } from '../types/badge';
+
+/**
+ * Badges migrated from [frontend implementation](https://github.com/guardian/frontend/blob/c52cc1d35cec5a0ec95075f5e2079595d26dc35f/common/app/model/Badges.scala)
+ * @todo There are much nicer ways to store this kind of data
+ */
+export const FRONTEND_BADGES: FEBadgeType[] = [
```

Comment:
Are they frontend badges? Or are they just a list of existing badges? 
They do come from Frontend in that we have brought them over from that codebase, but I don't think they necessarily should be named "frontend" as this can be confusing - specifically we usually prefix with "FE" (for FrontEnd) data that is actually sent to DCR from frontend in the (POST) request to render